creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -1194,35 +1195,44 @@ class SnapbackSM {
    * @notice this will issue sync to healthy secondary and update replica set away from unhealthy secondary
    */
   async aggregateReconfigAndPotentialSyncOps(nodeUsers, unhealthyPeers) {
+    // Parallelize calling this._aggregateOpsWithQueriedSpIds on chunks of 500 nodeUsers at a time
+    const nodeUserBatches = _.chunk(
+      nodeUsers,
+      AGGREGATE_RECONFIG_AND_POTENTIAL_SYNC_OPS_BATCH_SIZE
+    )
+    const resultBatches = []
+    while (nodeUserBatches.length) {
+      const nodeUserBatch = nodeUserBatches.shift()
+      const resultBatch = await Promise.allSettled(
```

Comment:
oh sweet, TIL about allSettled
historically we've used promise.all with internal try-catches to prevent halting but this is slick